app/mailers/mailer.rb

```diff
@@ -98,4 +98,15 @@ def notify_grade_conflict_message(defn)
     mail(subject: defn[:subject],
          to: defn[:to])
   end
+
+  #Email about a review rubric being changed. If this is successful, then the answers are deleted for a user's response
+  def notify_review_rubric_change(defn)
+    @body = defn[:body]
+    @answers = defn[:body][:answers]
+    @name = defn[:body][:name]
+    @assignment_name = defn[:body][:assignment_name]
+    defn[:to] = 'jwboerge@ncsu.edu' if Rails.env.development? || Rails.env.test?
```

Comment:
are we gonna change the to email to something else later? probably not a good idea to hardcode this if/when merging into main repo

Reply:
I agree, we should change this back to expertiza.development@gmail.com before the project is over

Reply:
Agreed. That was left in from my mailer testing. We can swap it back
```suggestion
    defn[:to] = 'expertiza.development@gmail.com' if Rails.env.development? || Rails.env.test?
```